packages/synapse-react-client/src/components/DownloadCart/RequestDownloadCard.test.tsx

```diff
@@ -13,14 +13,21 @@ import {
   ErrorResponse,
   FeatureFlagEnum,
 } from '@sage-bionetworks/synapse-types'
-import { render, screen, waitFor, within } from '@testing-library/react'
+import { act, render, screen } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
+import EntityAclEditorModal from '../EntityAclEditor/EntityAclEditorModal'
 import {
   REQUEST_DOWNLOAD_TITLE,
   RequestDownloadCard,
   RequestDownloadCardProps,
 } from './RequestDownloadCard'
 
+vi.mock('../EntityAclEditor/EntityAclEditorModal', () => ({
+  default: vi.fn(() => null),
+}))
+
+const mockEntityAclEditorModal = vi.mocked(EntityAclEditorModal)
```

Comment:
Test was flaky, easier to use a mock for the component where we were getting flaky behavior